File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/like/HeartCakeShopResponseParam.java
package com.cakk.domain.mysql.dto.param.like;

import java.util.Set;

import com.cakk.common.enums.Days;

public record HeartCakeShopResponseParam(
	Long cakeShopHeartId,
	Long cakeShopId,
	String thumbnailUrl,
	String cakeShopName,
	String cakeShopBio,
	Set<String> cakeImageUrls,
	Set<Days> operationDays
) {
}

File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/like/HeartCakeShopResponseParam.kt
package com.cakk.domain.mysql.dto.param.like

import com.cakk.common.enums.Days

data class HeartCakeShopResponseParam(
	val cakeShopHeartId: Long,
	val cakeShopId: Long,
	val thumbnailUrl: String,
	val cakeShopName: String,
	val cakeShopBio: String,
	val cakeImageUrls: Set<String>,
	val operationDays: Set<Days>
)
